plugins/modules/dns_auth_zone.py

```diff
@@ -630,12 +630,12 @@
 EXAMPLES = r"""
   - name: Create an Auth Zone
     infoblox.bloxone.dns_auth_zone:
-      name: "example_zone"
+      fqdn: "example_zone"
```

Comment:
Lets keep this PR for Delegation , I will raise a separate PR for this

Reply:
i have reverted the change